### PR TITLE
REPOSITORY : Add `User`

### LINKED ISSUE #18: REPOSITORY: Add `User`

### PR DESCRIPTION
**Pull request by Nabiddinov**
Closes #18

### CODE REVIEW

File: CountPad.Application/Services/UserService.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+using System.Threading.Tasks;
+using CountPad.Application.Interfaces.RepositoryInterfaces;
+using CountPad.Application.Interfaces.ServiceInterfaces;
+using CountPad.Domain.Models.Users;
+
+namespace CountPad.Application.Services
+{
+    public class UserService : IUserService
+    {
+        private IUserRepository userRepository;
```
Comment (MirzayevDeveloper) [inline CountPad.Application/Services/UserService.cs:15]: do readonly

File: CountPad.Application/Services/UserService.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+        public UserService(IUserRepository userRepository) =>
+           this.userRepository = userRepository;
+
+        public Task<int> AddUserAsync(User user)
```
Comment (MirzayevDeveloper) [inline CountPad.Application/Services/UserService.cs:20]: make like this
```csharp
public Task<int> AddUserAsync(User user) =>
    this.userRepository.AddAsync(user);
```

**Review by Ulugbek4499 — COMMENTED** [review]
Header File needed

File: CountPad.Application/Services/UserService.cs (hunk 1 above)
Comment (Nabiddinov) [reply CountPad.Application/Services/UserService.cs]: yes

File: CountPad.Application/Services/UserService.cs (hunk 2 above)
Comment (Nabiddinov) [reply CountPad.Application/Services/UserService.cs]: yes

**Review by MirzayevDeveloper — CHANGES_REQUESTED** [review]
resolve conflicts

**Pull request merged**